source/upgrade-guide/troubleshooting.rst

```diff
@@ -0,0 +1,104 @@
+
+
+No username and password found in the keystore
+----------------------------------------------
+
+For alerts and vulnerabilities detected by Vulnerability detection to be indexed and displayed on the dashboard, it is necessary to add the credentials of the indexer in the manager keystore. These credentials are those obtained at the time of installation.
+
+
+.. code-block:: console
+
+   # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_USERNAME>
+   # /var/ossec/bin/wazuh-keystore -f indexer -k password -v <INDEXER_PASSWORD>
+
+
+IndexerConnector initialization failed
+--------------------------------------
+
+This warning may be due to incorrect keystore credentials, as mentioned in the previous section, or there may be a configuration or certificate error. To solve this it is necessary to check that the :doc:`indexer section</user-manual/reference/ossec-conf/indexer>` is correctly configured in ``/var/ossec/etc/ossec.conf`` by checking that the IP, port and certificates path are as expected.
+
+Once the error has been fixed and it has been possible to connect to the indexer, we will be able to see a log like the following one:
+
+.. code-block:: console
+
+   INFO: IndexerConnector initialized successfully for index: ...
+
+If the error persists, temporarily enable ``wazuh_modules.debug=2`` in ``/var/ossec/etc/local_internal_options.conf`` to get more information on what may be causing the issue.
+
+
+Vulnerability detection seems to be disabled or has a problem
+-------------------------------------------------------------
+
+This warning shows that Vulnerability detection may be disabled or there is an error in the configuration. To check what may be happening, we must first check that ``vulnerability-detection`` is enabled in ``/var/ossec/etc/ossec.conf``. If it is enabled, we have to check that the vulnerability index ``wazuh-states-vulnerabilities-*`` has been created correctly. This can be checked in ``Indexer Management > Index Management > Indices``.
+
+If the index has not been created, check the manager logs for any errors or warnings, as this may be due to an error mentioned in the previous sections:
+
+.. code-block:: console
+
+    # cat /var/ossec/logs/ossec.log | grep -i -E "error|warn"
+
+
+Application Not Found
+---------------------
+
+If after updating, when accessing the dashboard we find the message ``Application Not Found``, this may be because after the update the file ``/etc/wazuh-dashboard/opensearch_dashboards.yml`` has not been overwritten with the new changes. To fix this, please add the following line to the configuration file:
```

Comment:
```suggestion
If after updating, when accessing the dashboard we find the message ``Application Not Found``, it is possible that the ``/etc/wazuh-dashboard/opensearch_dashboards.yml`` configuration file has not been overwritten with the new changes after the upgrade. To fix this, replace the following line in the configuration file:
```

Reply:
```suggestion
If you encounter the message *Application Not Found* when accessing the Wazuh dashboard after upgrading, it might be that the configuration file ``/etc/wazuh-dashboard/opensearch_dashboards.yml`` wasn't overwritten with new changes. To resolve this issue, replace the following line in the configuration file:
```